src/pages/tabs/tabs.ts

```diff
@@ -32,4 +32,4 @@ export class TabsPage {
   ngOnDestroy() {
     this.itemSubs.unsubscribe(); //???
   }
-}
\ No newline at end of file
+}
```

Comment:
On ngOnInit function: you get cartItems and then assign it to cartNum (Should get res.length)